libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,20 +1,36 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
-  <div class="reading-list-item" *ngFor="let book of readingList$ | async">
+  <div class="reading-list-item" *ngFor="let item of readingList$ | async">
     <div>
-      <img class="reading-list-item--cover" [src]="book.coverUrl"  alt="Book Cover"/>
+      <img class="reading-list-item--cover" alt="book cover" [src]="item.coverUrl" />
```

Comment:
For non-interactive element, we can keep `alt=""`

Reply:
changed alt value